### ISSUE TITLE
Incompatible with SQLAlchemy 2.0.0

### ISSUE THREAD

**Issue by Zlopez** [fschulze/sqlalchemy_schemadisplay#33]
Today I got a automatic [PR](https://github.com/fedora-infra/anitya/pull/1534) updating SQLAlchemy to 2.0.0 and the sqlalchemy_schemadisplay stopped working.

Here is the backtrace:
```
Traceback (most recent call last):
  File "/var/home/zlopez/git/anitya/docs/generate_db_schema", line 29, in <module>
    write_graph('images/database.png')
  File "/var/home/zlopez/git/anitya/docs/generate_db_schema", line 18, in write_graph
    graph = create_schema_graph(
  File "/var/home/zlopez/git/anitya/.tox/docs/lib/python3.10/site-packages/sqlalchemy_schemadisplay.py", line 153, in create_schema_graph
    label=_render_table_html(table, metadata, show_indexes, show_datatypes),
  File "/var/home/zlopez/git/anitya/.tox/docs/lib/python3.10/site-packages/sqlalchemy_schemadisplay.py", line 121, in _render_table_html
    if metadata.bind and isinstance(metadata.bind.dialect, PGDialect):
AttributeError: 'MetaData' object has no attribute 'bind'
```

The whole script for generation database schema could be found [here](https://github.com/fedora-infra/anitya/blob/5e9d72283de37a4904f4a66a1d5b2a1454623ead/docs/generate_db_schema).

**Comment by return42** [comment]
I have the same issue .. BTW are there any plans to build a new release / [on pypi there is v1.3 - 2016-01-27](https://pypi.org/project/sqlalchemy_schemadisplay/)

**Comment by Zlopez** [comment]
This is fixed in 2.0 release.
